rasa/core/run.py

```diff
@@ -195,32 +195,3 @@ def serve_application(core_model=None,
                                action_endpoint=endpoints.action)
 
     return app.agent
-
-
-if __name__ == '__main__':
```

Comment:
I guess we should do the same here as we did it in server.py and raise a runtime error with a link to the migration guide